analyzer/libretime_analyzer/message_listener.py

```diff
@@ -174,8 +174,8 @@ def spawn_analyzer_process(
             )
             metadata = queue.get()
         except Exception as exception:
-            logger.error("Analyzer pipeline exception: %s" % str(exception))
-            metadata["import_status"] = PipelineStatus.failed
+            logger.error(f"Analyzer pipeline exception: {exception}")
```

Comment:
Maybe rather log the message and then the exception as logger.exception to match what is in #2027?